core/lib/config/src/constants/ethereum.rs

```diff
@@ -12,8 +12,8 @@ pub const GUARANTEED_PUBDATA_PER_L1_BATCH: u64 = 4000;
 
 /// The maximum number of pubdata per L1 batch. This limit is due to the fact that the Ethereum
 /// nodes do not accept transactions that have more than 128kb of pubdata.
-/// The 18kb margin is left in case of any inpreciseness of the pubdata calculation.
-pub const MAX_PUBDATA_PER_L1_BATCH: u64 = 120000;
+/// The 128kb margin is left in case of any inpreciseness of the pubdata calculation.
+pub const MAX_PUBDATA_PER_L1_BATCH: u64 = 110000;
```

Comment:
1. Here the margin left is 18kb and not 128kb 
2. Why do you think such change is needed? We should generally be able to send transactions with size up to 128kb. Some margin is needed, but why that large?